packages/angular/cli/utilities/log-file.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { appendFileSync, mkdtempSync, realpathSync } from 'fs';
+import { tmpdir } from 'os';
+import { normalize } from 'path';
+
+let logPath: string | undefined;
+
+/**
+ * Writes an Error to a temporary log file.
+ * If this method is called multiple times from the same process the same log file will be used.
+ * @returns The path of the generated log file.
+ */
+export function writeErrorToLogFile(error: Error): string {
```

Comment:
This file looks like it was erroneously included because it was part of another branch.

Reply:
Indeed